index.ts

```diff
@@ -18,6 +18,7 @@ declare namespace ethereum {
 /** Host IPFS interface */
 export declare namespace ipfs {
   function cat(hash: string): Bytes | null
+  function map(hash: String, callback: String, flags: String[]): void
```

Comment:
This is fine, but I'd add a
```
function mapJSONLines(hash: String, callback: String): void
```
here and add a host export for it in graph-node. It _may_ work to add something like
```
namespace ipfs {
  function mapJSONLines(hash: String, callback: String): void {
    ipfs.map(hash, callback, ["json"])
  }
}
```
here in `index.ts`  without requiring an extra host export in graph-node. Worth a shot.

Reply:
As we discussed in the graph node PR, it's best not to have the name refer to JSON lines.

Reply:
I added a `mapJSON` alias.